### ISSUE TITLE
Behörighet att rätta uppgifter i Studentportalen

### ISSUE THREAD

**Issue by kamar535** [simonlovgren/introit-16#88]
AKKA-ansvarig meddelar:

> Har nu lagt in alla på listan i AKKA utom Erik Ekstedt och Andreas Widmark vars personnummer inte är korrekt inskrivna på listan och jag vill inte gissa.
> Alla har fått slutdatum 2016-11-30 inlagt. De behöver aktivera sina nya konton som är hopkopplade med e-postadresserna.
> 

Hela ärendet: http://www.it.uu.se/internt/web/bugzilla/show_bug.cgi?id=9344
Jag har kompletterat med personnummer för Ekstedt och Widmark. 

Varje handledare behöver nu besöka https://konto.weblogin.uu.se/ och logga in (Joint Web Login) för att aktivera sin nya roll som övrigt verksam.

**Comment by kamar535** [comment]
Efter aktivering behöver jag (Karl) koppla varje enskild handledare till respektive kurs i Studentportalen.

För att jag (Karl) skall kunna göra detta behöver jag få behörighet som behörighetsadministratör på samtliga berörda institutioner. Jag inväntar fortfarande underskrift på beslut från kanslichef Måns Östring vid Områdeskanslier, Kansliet för naturvetenskap och teknik: http://katalog.uu.se/empinfo/?id=N13-1814

**Comment by kamar535** [comment]
Nu är Erik Ekstedt och Andreas Widmark ochså inlagda som övrigt verksamma i AKKA.

**Comment by simonlovgren** [comment]
Jag lägger med info i det mailutskick jag ska fixa ikväll.

**Comment by kamar535** [comment]
@simonlovgren @Kurogamu om ni inte redan gjort det, kan ni aktivera er nya behörighet som externt verksam?

Jag vill testa hur det fungerar att lägga till er som lärarassistenter på samtliga kurser som berörs av Introduktion till datorer och ser för tillfället inte era namn bland de jag kan lägga till.

**Comment by simonlovgren** [comment]
@kamar535 nu har jag aktiverat mitt konto. Ber om ursäkt för fördröjningen

**Comment by kamar535** [comment]
@simonlovgren Jag har nu kopplat kontot `simlo254` med rollen **lärarassistent** på följande kurser i studentportalen:

1TE705	
1TE609
1DT051	
1MA219	
1TE605	
1TE610	
5LH220	
1TV000

Tänk på att du måste logga in med  `simlo254`. Testa gärna för att se hur det ser ut när du loggar in och  skriv en kort guide för övriga handledare hur det går till väga för att logga in och rätta uppgifter.

**Comment by kamar535** [comment]
@simonlovgren  Allt eftersom du får bekräftat vilka handledare som aktiverat sina nya konton, uppdatera kolumnen **Aktiverat** här https://docs.google.com/spreadsheets/d/1nMCeIvNau7xWTFJ4AEja5L2x-aaJrQuzYgJ28w4WHQs/edit#gid=562541617

**Comment by simonlovgren** [comment]
Testat att logga in och kan verifiera att jag kommer åt samtliga kurser via Utbildning > Kurstillfällen > Alla

**Comment by simonlovgren** [comment]
@kamar535 hur är det man gör för att rätta i studentportalen? Jag ser att det finns en filarea per uppgift där man kan hämta uppgifterna som laddats upp. När jag kollar i framsteg och klickar i ex. studenttjänster för en student får jag upp en drop-down och en textruta samt en checkruta (kompletterad), men det finns inga alternativ i dropdownen. Är detta då det inte finns någonting inlämnat för den studenten ännu eller skall det redan nu finnas alternativ att välja?

**Comment by Kurogamu** [comment]
Nu har jag också aktiverat!

Vänliga hälsningar,
Daniel

**Comment by kamar535** [comment]
@simonlovgren För mig ser det ut så här på kursen 1DT051 och inlämning Studenttjänster.

![screen shot 2016-08-30 at 10 23 24](https://cloud.githubusercontent.com/assets/8029215/18081731/e0a47586-6e9b-11e6-963b-7e0a3eef5b5f.png)

Här ser vi t.ex. att Annerwall, Staffan lämnat in en fil som går att ladda hem direkt genom att klicka på ikonen i kolumnen filer. Sedan kan jag sätta **status** för inlämningen i rullgardinen i kolumnen **status**. Vid direkt godkännande skall status sättas till **klar**. Vid komplettering skall status sättas till ** Rest - kompletteras**. Vid godkännande efter komplettering skall status sättas till **Klar efter komplettering**.

**Comment by simonlovgren** [comment]
Ok, så då ändrar vi/handledarna endast där och det kommer automatiskt visas som klar/komplettering/etc. Under framsteg?

**Comment by kamar535** [comment]
Ja, det är så det är skall fungera, dvs en automatisk koppling av inlämning och framsteg. Prova att rätta en uppgift (Staffan Annerwall) och ändra status och verifiera att status i framsteg "hänger med".

**Comment by kamar535** [comment]
@Kurogamu Jag har nu kopplat kontot **danah407** med rollen **lärarassistent** på följande kurser i Studentportalen:

- 1TE705
- 1TE609
- 1DT051
- 1MA219
- 1TE605
- 1TE610
- 5LH220
- 1TV000

Tänk på att du måste logga in med **danah407**  i Studentportalen när du rättar inlämnade uppgifter.

**Comment by Kurogamu** [comment]
Japp, det ser ut att fungera finfint!

2016-08-30 8:41 GMT+00:00 Karl Marklund <notifications@github.com>:

**Comment by Kurogamu** [comment]
Hej!
Kan du lägga till behörighet för Sahand Shamal (sahsh208)?
Simon har lagt till honom i driven.

**Comment by kamar535** [comment]
@Kurogamu fixat behörighet för Sahand.
